src/components/Authentication/LoginPage.js

```diff
@@ -56,22 +75,44 @@ const LoginPage = () => {
     const [getUserType] = useLazyQuery(GET_USER_TYPE, {
         variables: { username: email },
         onCompleted: (data) => {
-            setUserType(data.userType);
+            setUserType(data?.userType?.userType);
+            setGoogleAuthEnabled(data?.userType?.googleAuthEnabled);
             if (userType === null) {
                 setHasError(true);
             }
         },
     });
+    const [getVerifyGoogleOauthTokenStatus] = useLazyQuery(
+        VERIFY_GOOGLE_OAUTH,
+        {
+            variables: { loginEmail: email, oauthEmail: googleAuthEmail },
+            onCompleted: async (data) => {
+                setVerifyGoogleOauthTokenStatus(
+                    data?.verifyGoogleOauth?.verified
```

Comment:
why would data be undefined here? if it is, then you should do a larger overall check and wrap the whole thing in an if statement